safe/gis/vector/summary_5_multi_exposure.py

```diff
@@ -157,6 +158,14 @@ def multi_exposure_analysis_summary(analysis, intermediate_analysis):
     analysis.commitChanges()
     analysis.keywords['title'] = layer_purpose_analysis_impacted['name']
     analysis.keywords['layer_purpose'] = layer_purpose_analysis_impacted['key']
+
+    # Set up the extra keywords so everyone knows it's a
+    # multi exposure analysis result.
+    extra_keywords = {
+        'analysis_type': MULTI_EXPOSURE_ANALYSIS_FLAG
```

Comment:
Can you create the entry for `analysis_type` in the definition instead and use it where you need to read this entry.